### PR TITLE
GameDB: fixes for various games 

### PR DESCRIPTION
**Pull request by Mrlinkwii**
### Description of Changes

fixes for various games
### Rationale behind Changes

fixes: #10156
### Suggested Testing Steps

watch CI

### CODE REVIEW

File: bin/resources/GameIndex.yaml [hunk 1]
```diff
@@ -12664,6 +12664,12 @@ SLES-50212:
 SLES-50213:
   name: "NFL Quarterback Club 2002"
   region: "PAL-E"
+  patches:
+    F64D1146:
+      content: |-
+        //Fix freezes caused by sceipusync
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:12670]: space after // and period at end

File: bin/resources/GameIndex.yaml [hunk 2]
```diff
@@ -15621,6 +15627,8 @@ SLES-51459:
 SLES-51460:
   name: "Football Manager 2004"
   region: "PAL-I"
+  clampModes:
+    vu1ClampMode: 3 # Fixes 3d rendering.
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:15631]: 3D is capitalised

File: bin/resources/GameIndex.yaml [hunk 3]
```diff
@@ -16836,6 +16844,8 @@ SLES-52021:
 SLES-52022:
   name: "Total Club Manager 2004"
   region: "PAL-M4"
+  clampModes:
+    vu1ClampMode: 3 # Fixes 3d rendering.
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:16848]: 3D is capitalised

File: bin/resources/GameIndex.yaml [hunk 4]
```diff
@@ -18373,6 +18383,11 @@ SLES-52673:
 SLES-52674:
   name: "Fussball Manager 2005"
   region: "PAL-M5"
+  clampModes:
+    vu1ClampMode: 3 # Fixes 3d rendering.
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:18387]: 3D is capitalised

File: bin/resources/GameIndex.yaml [hunk 5]
```diff
@@ -55451,6 +55466,12 @@ SLUS-20153:
 SLUS-20154:
   name: "NFL Quarterback Club 2002"
   region: "NTSC-U"
+  patches:
+    F3773B92:
+      content: |-
+        //Fix freezes caused by sceipusync
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:55472]: space after // and period at end

**Review by JordanTheToaster — APPROVED** [review]

**Pull request merged**